components.tsx

```diff
@@ -272,11 +310,11 @@ const Tab = (props: {
 
 export {
```

Comment:
Btw, I really dislike this way of exporting components, there are 2 reasons why this is bad:
1. You cannot tree-shake the dependencies (not super important for this project, but for libraries it is) https://medium.com/@netxm/what-is-tree-shaking-de7c6be5cadd
2. You cannot see if you've remembered to export the component you're looking at without scrolling to the bottom.

Reply:
I think the second point is very sound, and I will update the component file.
But tree-shaking should work no?

    It only works with import and export. It won’t work with CommonJS require syntax. Same applies to npm dependencies.

I use export {...} so tree shaking should work?

Reply:
I don't know, it could, but from a little googling it seems that it's a more difficult problem to solve, so you'd have to look at what tree-shaker you're using